netbox/netbox-housekeeping-cronjob.yaml

```diff
@@ -0,0 +1,116 @@
+---
+# Source: netbox/templates/cronjob.yaml
```

Comment:
even for one yaml file I'd prefer helm template rather than raw yaml — labels, selectors, values are simpler to manage in the helm chart (_not to lose something_)

Reply:
I agree here, this is just a temporary "hack" as I ran into an issue with gke autopilot's automatic pod resource/limit admission webhook, I get this error when trying to perform an install: 
# helm install above fails when housekeeping is on with: admission webhook "workload-defaulter.common-webhooks.networking.gke.io" denied the request: no kind "CronJob" is registered for version "batch/v1" in scheme "pkg/runtime/scheme.go:100"

What happens is the upstream helm chart is changing the `apiVersion` to `batch/v1` because of the kubernetes version: https://github.com/bootc/netbox-chart/blob/5634761ad2d2ef33387154eec0efdc6f5163eb22/templates/cronjob.yaml#L2

and the admission webhook doesn't understand "batch/v1" apiVersions yet.
I need to report this as a bug to Google in some way to that we can get rid of this hack :)

Reply:
PS I got this object by running `helm template` with the same values as the install except for enabling housekeeping. 
Then stripping out all the the other resources except for this one, then just have garden perform a `kubectl apply` on this yaml.